docs/provision.adoc

```diff
@@ -0,0 +1,52 @@
+[[custom-runtime-connector-service]]
+= Custom Runtime Connector
+
+The Custom Runtime Connector (CRC) allows an existing custom service to be exposed to a mobile application.
+This creates a config map, which contains the service's configuration which is then exposed to the mobile application.
+
+This can be seen in the project's resources within the Openshift web console or by running the following command using the mobile CLI.
+
+```bash
+oc plugin mobile get clientconfig <clientID>
+```
+Where the _clientID_ (<clientName>-<clientType>) parameter is specified by the user.
```

Comment:
Can we add some docs or a link to show how the user would retrieve the client id here?